resources/monitoring/values.yaml

```diff
@@ -516,18 +516,22 @@ grafana:
   ## Configure additional grafana datasources
   ## ref: https://grafana.com/docs/grafana/latest/features/datasources/
   additionalDataSources:
-    - name: Loki
-      type: loki
-      access: proxy
-      url: http://logging-loki.kyma-system:3100
-      editable: true
-      jsonData:
-        maxLines: 1000
-    - name: Jaeger
-      type: jaeger
-      access: proxy
-      url: http://tracing-jaeger-query.kyma-system:16686
-      editable: true
+    LokiEnabled: true
```

Comment:
See comment above, should be a separate key outside of additionalDataSources.
However, you should use keys with first letter lowerCase and use nested structure like in:
loki:
  enabled: true
  config:
     ...